Sources/SRGLetterbox/SRGLetterboxService~ios.m

```diff
@@ -579,7 +579,7 @@ - (void)updateMetadataWithController:(SRGLetterboxController *)controller
 - (NSURL *)artworkURLForController:(SRGLetterboxController *)controller withWidth:(SRGImageWidth)width
 {
     SRGMedia *media = controller.displayableMedia;
-    NSURL *artworkURL = SRGLetterboxArtworkImageURL(media.image, width, controller);
+    NSURL *artworkURL = SRGLetterboxImageURL(media.image, width, controller);
```

Comment:
Issue with the implementation clearly visible here. The method was changed but the new one expects a size, not a width (both are integer-based enums so the compiler does not warn about the mismatch for some reason).

The result is that artwork image URLs are wrong. You should therefore either:

- Keep both width- and size-based APIs.
- Or use a size close to what is expected for the artwork (e.g. `SRGImageSizeMedium`) not a width.

Reply:
Thanks a lot @defagos for the analyse and propositions. UI tests as no unit tests.

Artwork image has now switched to `SRGImageSizeMedium` and `SRGImageVariantDefault`.